### PR TITLE
API documentation for v1

### PR DESCRIPTION
**Pull request by drumilpatel2000**
Signed-off-by: Drumil Patel <drumilpatel720@gmail.com>

Add API documentation for `api/v1`

### CODE REVIEW

**Comment by drumilpatel2000** [comment]
@beorn7 Please have a look

File: README.md [hunk 1]
```diff
@@ -392,6 +392,110 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
 * For example to wipe all metrics from the Pushgateway:
 
         curl -X PUT http://pushgateway.example.org:9091/api/v1/admin/wipe
+
+## Rest API
```
Comment (beorn7) [inline README.md]: This API is not really RESTful, or at least the push API above is more REST-like than this one.

How about _Query API_?

File: README.md [hunk 2]
```diff
@@ -392,6 +392,110 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
...
+
+Rest API provides Rest based interface for accessing metrics and build information.
```
Comment (beorn7) [inline README.md]: Then this would read:

The query API allows accessing pushed metrics and build and runtime information.

File: README.md [hunk 3]
```diff
@@ -392,6 +392,110 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
...
+### URL
+
+    /api/<API_VERSION>/<HANDLER>
+    
+ * Available endpoints:
+ 
+| HTTP_METHOD| API_VERSION |  HANDLER | DESCRIPTION |
+| :-------: |:-------------:| :-----:| :----- |
+| GET     | v1 | status |  Returns build information in JSON format. |
+| GET     | v1 | metrics |  Returns information of families of metrics in JSON format. |
```
Comment (beorn7) [inline README.md]: How about: "Returns the pushed metric families in JSON format."

File: README.md [hunk 4]
```diff
@@ -392,6 +392,110 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
...
+
+### URL
+
+    /api/<API_VERSION>/<HANDLER>
+    
+ * Available endpoints:
+ 
+| HTTP_METHOD| API_VERSION |  HANDLER | DESCRIPTION |
+| :-------: |:-------------:| :-----:| :----- |
+| GET     | v1 | status |  Returns build information in JSON format. |
```
Comment (beorn7) [inline README.md]: "Returns build information, command line flags, and the start time in JSON format."

**Comment by beorn7** [comment]
Thanks.

I guess we are ready for a release. I'll cut one today.

**Pull request merged**